CHANGELOG.md

```diff
@@ -28,6 +28,14 @@
   * Fixed conditional logic for creation and update.
 * IntuneTrustedRootCertificateIOS
   * Initial release
+* IntuneVPNConfigurationPolicyAndroidDeviceOwner
+  * Initial release
+* IntuneVPNConfigurationPolicyAndroidEnterprise
+  * Initial release
+* M365DSCDRGUtil
```

Comment:
This was part of a PR from @FabienTschanz which was already merged, please remove these 2 entries from the changelog

Reply:
removed 

* IntuneVPNConfigurationPolicyAndroidEnterprise
  * Initial release

Reply:
No, that's wrong. Add back your entry, what you need to remove is M365DSCDRGUtil/M365DSCUtil entries

Reply:
Ok, not sure how they ended up as part of my commit, but I've removed them! Hopefully done the right thing this time?

Reply:
yeah looks ok now :)